charts/rmd/values.yaml

```
@@ -0,0 +1,114 @@
+    initialDelaySeconds: 60
+    # rmd.livenessProbe.periodSeconds -- How often (in seconds) to perform the probe
+    periodSeconds: 10
+    # rmd.livenessProbe.timeoutSeconds -- Number of seconds after which the probe times out
+    timeoutSeconds: 5
+    # rmd.livenessProbe.failureThreshold -- Number of tries to perform the probe
+    failureThreshold: 6
+    # rmd.livenessProbe.successThreshold -- Minimum consecutive successes for the probe to be considered successful after having failed
+    successThreshold: 1
+  readinessProbe:
+    # rmd.readinessProbe.enabled -- Enable readiness probe on rmd
+    enabled: true
+    # rmd.readinessProbe.initialDelaySeconds -- Number of seconds after the container has started before readiness probe is initiated
+    initialDelaySeconds: 30
+    # rmd.readinessProbe.periodSeconds -- How often (in seconds) to perform the probe
+    periodSeconds: 10
+    # rmd.readinessProbe.timeoutSeconds -- Number of seconds after which the probe times out
+    timeoutSeconds: 5
+    # rmd.readinessProbe.failureThreshold -- Number of tries to perform the probe
+    failureThreshold: 6
+    # rmd.readinessProbe.successThreshold -- Minimum consecutive successes for the probe to be considered successful after having failed
+    successThreshold: 1
+  settings:
+    # django settings for rmd
+    # rmd.settings.emailHost -- Connection string of SMTP server to send mails
+    emailHost: smtp.syhosting.ch:587
+    # rmd.settings.emailFrom -- Default email address to use for various responses
+    emailFrom: maildelay@rmd.io
+    # rmd.settings.serverEmail -- Email address error messages are sent from
+    serverEmail: maildelay@rmd.io
+  cronjobs:
+    # rmd.cronjobs.importMails -- import Mails
+    importMails:
+      schedule: "0 * * * *"
+      command:
+        - "app/manage.py"
+        - "import"
+    # rmd.cronjobs.sendMails -- send Mails
+    sendMails:
+      schedule: "10 * * * *"
+      command:
+        - "app/manage.py"
+        - "sendmail"
+
+ingress:
+  # ingress.enabled -- Enable ingress for rmd
+  enabled: false
+  # ingress.annotations -- Ingress annotations
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  # ingress.hosts -- Ingress hostnames
+  hosts: []
+  #  - rmd-test.example.com
+  # ingress.tls -- Ingress TLS options
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - rmd-test.example.com
+
+postgresql:
+  # postgresql.enabled -- Enable PostgreSQL for persistence
+  enabled: true
+  # postgresql.postgresqlDatabase -- PostgreSQL database name
+  postgresqlDatabase: rmd.io
+  # postgresql.postgresqlUsername -- PostgreSQL user name
+  postgresqlUsername: postgres
+  # postgresql.postgresqlPassword -- Set a password for PostgreSQL
+  # postgresqlPassword: s3cr3t
+  image:
+    # postgresql.image.tag -- PostgreSQL image version to use
+    tag: 12.2.0
```

Isn't 12.3 the latest version?
```suggestion
    tag: 12.3.0
```